Dockerfile.arm32v7

```diff
@@ -1,12 +1,13 @@
-FROM alpine AS builder
+FROM alpine as qemu
 
-# Download QEMU, see https://github.com/docker/hub-feedback/issues/1261
-ENV QEMU_URL https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-aarch64.tar.gz
-RUN apk add curl && curl -L ${QEMU_URL} | tar zxvf - -C . --strip-components 1
+ARG QEMU_VERSION "v4.2.0-7"
```

Comment:
@nightah  `ENV QEMU_VERSION "v4.2.0-7"` ?

Reply:
Got a: 
```
Unexpected error
Encountered error: 400 Client Error: Bad Request ("Dockerfile parse error line 3: ARG requires exactly one argument")
```

Reply:
No an ARG is fine it doesn't need to persist beyond the download.

Reply:
https://vsupalov.com/docker-arg-vs-env/ just FYI.

Reply:
So might be `ARG QEMU_VERSION="v4.2.0-7" to solve the parse error ? (https://docs.docker.com/engine/reference/builder/#default-values)

Reply:
Yep the above is correct syntax, if you can correct it that'd be great thanks.